mkdocs/docs/development.md

```
@@ -136,6 +136,18 @@ use the following commands on PRs to interact with it:
 
 - `@coderabbitai help` - Displays available commands and usage information.
 
+#### Gource
+
+1. Install Gource:
+   ```bash
+   sudo apt-get update
+   sudo apt-get install --yes gource ffmpeg
+   ```
+2. Generate the video:
+   ```bash
```

Markdownlint (0.37.0)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)